README.md

```diff
@@ -1,7 +1,7 @@
 # KubeVirt
 
 <p align="center">
-<img src="https://cncf-branding.netlify.app/img/projects/kubevirt/icon/color/kubevirt-icon-color.png" width="100">
```

Comment:
Link is not validate anymore

https://github.com/cncf/artwork/blob/master/projects/kubevirt/icon/color/kubevirt-icon-color.png is an alt